homeassistant/components/netgear/router.py

```diff
@@ -200,6 +201,13 @@ async def async_get_traffic_meter(self) -> None:
         async with self._api_lock:
             return await self.hass.async_add_executor_job(self._api.get_traffic_meter)
 
+    async def async_allow_block_device(self, mac, allow_block) -> None:
```

Comment:
Please type the whole signature when adding type annotations.

Reply:
I added the : str types